packages/Card/src/components/Text/Text.styles.js

```diff
@@ -7,5 +7,5 @@ export const textStyles = styled.p`
   ${stylers.fontSize(-1)};
   margin: 0;
   min-height: 59px;
-  padding: ${stylers.spacer(2)} 0;
+  padding: ${tokens.space};
```

Comment:
This should include `0` padding for left and right, otherwise you get some indentation on the `Text` block:
![Screen Shot 2021-12-07 at 1 25 27 PM](https://user-images.githubusercontent.com/14944896/145108889-7bebde2e-c414-4465-beaa-377e528f8f2f.png)

Reply:
Also, I still think this one can also be `margin` instead of `padding`. Did you try that already though?